tools/kagami/README.md

```diff
@@ -190,3 +191,22 @@ The output should be identical to the [reference configuration](../../docs/sourc
     ```bash
     kagami config client > client-config.json
     ```
+
+## `examples`
+To see which use cases you can choose from, run the following command:
+```bash
+kagami examples --help
+```
+
+- Generate an instruction as JSON that creates a new account:
+    ```bash
+    kagami examples create-account
+    ```
+- Generate an instruction as JSON that mints a public key:
+    ```bash
+    kagami examples mint-public-key --account-id "bob@wonderland" --public-key "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
```

Comment:
There was https://github.com/hyperledger/iroha/pull/3316, which transformed all keys in the repo to a new format: lowercase multihash plus uppercase hex. Shouldn't it be same here too?

Reply:
should be I guess